### ISSUE TITLE
Fix duplicate time stamps when reshuffling swath files

### ISSUE THREAD

**Issue by wpreimes** [TUW-GEO/repurpose#41]
Ts2Img from swath files can lead to duplicate time stamps in the time series at the transition from one swath to another when spatial resampling is performed (no fixed grid). 

The easiest solution would be to set a thereshold for a minimum time difference (e.g `t_min=300` seconds) to filter out duplicate time stamps whenn writing down time series (should also be checked across subsequent time chunks).

@sebhahn does that sound like a solution to your problem? Is there any case when you can have multiple measurements in a point that are less than e.g. 300 seconds aparat?

**Comment by sebhahn** [comment]
yes a final filter checking for "close" duplicates makes sense (perhaps also showing a warning to the user). choosing first/last could be tricky if a certain field (e.g. soil moisture) is important and you want to choose the observation which is not NaN (if any)
